metricbeat/helper/prometheus/prometheus.go

```diff
@@ -142,16 +155,31 @@ func (p *prometheus) GetProcessedMetrics(mapping *MetricsMapping) ([]common.MapS
 			// Convert labels
 			labels := common.MapStr{}
 			keyLabels := common.MapStr{}
+
```

Comment:
is this intended?

Reply:
you are right, for consistency that blank line shouldn't be there.
thanks